app/views/users/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is the info for a particular post</h1>
```

Comment:
- [ ] It would be better if we create a link `See all post` which when clicked should redirect us to the user's post index page as seen in the wireframe